runtime-modules/content/src/tests/mock.rs

```diff
@@ -1,7 +1,7 @@
 #![cfg(test)]
 
+use crate::sp_api_hidden_includes_decl_storage::hidden_include::traits::Currency;
```

Comment:
It looks like an auto-added line. `hidden_include` is non-documented export. We should use `Currency` trait from the frame_support crate.